src/components/views/lobby/Lobby.tsx

```diff
@@ -45,11 +45,13 @@ const Lobby = observer(() => {
             <div className="div-rooms">
               <div className="roomlist">
                 <TransitionGroup>
-                  {games.map((game) => (
-                    <Collapse key={game._links.self.href}>
-                      <RoomRow game={toJS(game)} />
-                    </Collapse>
-                  ))}
+                  {games
+                    .filter((game) => game.gameState === "FINDING_PLAYERS")
```

Comment:
i would compare the string with the enum (and maybe filter before and save the result in a variable with a speaking name